litmus-portal/clusterwide-k8s-manifest.yml

```diff
@@ -64,54 +151,79 @@ spec:
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: DataBaseServer
             - name: JWT_SECRET
-              value: "litmus-portal@123"
-            - name: EXTERNAL_ADDRESS
-              value: ""
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: JWTSecret
             - name: SERVICE_ADDRESS
               value: "http://litmusportal-server-service:9002"
             - name: SELF_CLUSTER
               value: "true"
             - name: SUBSCRIBER_IMAGE
               value: "litmuschaos/litmusportal-subscriber:ci"
-            - name: SUBSCRIBER_NAMESPACE
-              value: "litmus"
+            - name: AGENT_SCOPE
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: AgentScope
+            - name: AGENT_NAMESPACE
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: AgentNamespace
             - name: DEPLOYER_IMAGE
               value: "litmuschaos/litmusportal-self-deployer:ci"
-            - name: DEPLOYER_NAMESPACE
-              value: "litmus"
+            - name: ARGO_SERVER_IMAGE
+              value: "argoproj/argocli:v2.9.3"
+            - name: ARGO_WORKFLOW_CONTROLLER_IMAGE
+              value: "argoproj/workflow-controller:v2.9.3"
+            - name: ARGO_WORKFLOW_EXECUTOR_IMAGE
+              value: "argoproj/argoexec:v2.9.3"
+            - name: LITMUS_CHAOS_OPERATOR_IMAGE
+              value: "litmuschaos/chaos-operator:1.8.2"
+            - name: LITMUS_CHAOS_RUNNER_IMAGE
+              value: "litmuschaos/chaos-runner:1.8.2"
           ports:
-          - containerPort: 8080
+            - containerPort: 8080
           imagePullPolicy: Always
         - name: auth-server
           image: litmuschaos/litmusportal-auth-server:ci
           env:
             - name: DB_SERVER
-              value: "mongodb://mongo-service:27017"
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: DataBaseServer
             - name: JWT_SECRET
-              value: "litmus-portal@123"
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: JWTSecret
             - name: ADMIN_USERNAME
               value: "admin"
             - name: ADMIN_PASSWORD
               value: "litmus"
           ports:
-          - containerPort: 3000
+            - containerPort: 3000
           imagePullPolicy: Always
-      serviceAccountName: litmus-svc-account
+      serviceAccountName: litmus-server-account
 ---
 apiVersion: v1
 kind: Service
 metadata:
   name: litmusportal-server-service
   namespace: litmus
 spec:
-  type: NodePort
+  type: ClusterIP
```

Comment:
Why clusterIP?

Reply:
With NodePort for some reason Minikube testing failed, it was not able to find the service... should i change it back? @rajdas98

Reply:
Yes, Otherwise external cluster support will not work